### ISSUE TITLE
Migration to Android X

### ISSUE THREAD

**Issue by jaumard** [rmtmckenzie/flutter_qr_mobile_vision#69]
All flutter packages has moved to Android X and this one is causing warnings (maybe errors not sure...)

**Comment by jaumard** [comment]
I can confirm that it totally prevent people to migrate to androidX :( doesn't compile anymore

**Comment by jaumard** [comment]
Hey @rmtmckenzie are you still maintaining this repo ?

**Comment by rmtmckenzie** [comment]
Hi @jaumard sort of =D I've just been really busy, followed by getting really sick, and moving twice in the last little while. I don't know that I'll get to it today but I need to do some updates and look through PR's for the repo either today or tomorrow.

**Comment by jaumard** [comment]
Thanks for the reply @rmtmckenzie hope you feeling better now ! I'm asking this because I'm using your library at work, and some issues are blocking us like this one (thanks for the merge !) and the #64 I don't mind trying to fix it but I need to be sure I'll not doing it for nothing ^^ or it will be better to search another library to replace this one.

If you need help maintaining the repo I don't mind helping you with it

**Comment by rmtmckenzie** [comment]
Thanks, I appreciate the offer and I'll try to do better with PRs and whatnot so if there's anything you'd like to look into go for it =). But I'm just merging all the changes and I'll publish an updated version today, I've decided to spend a couple hours this morning and get some of the issues people are seeing sorted out.  I've merged the PRs that make sense to merge right now and I'll look into #64 to see if I can sort it out.

Part of the problem is that I started working on a larger update that made some pretty fundamental changes to how the entire plugin works (i.e. separate the 'controller' from the UI view, abstract and split out the plugin part so that either Firebase or Google Mobile Vision can be used, better control over resolution/flash/etc) and have been sort of holding off pushing some of these other updates until I finished with that - but it has turned out to be quite a lot of work and then I wasn't able to get anything done for a while. I don't think I'll have time to finish that today but I'm hoping sometime this week I'll get that larger update pushed out as well.
